File: MODUL 3/C/PRAK304-2310817320006-Alya_Rosaan.c
#include<stdio.h>

int main(){
    int a;
    scanf("%d", &a);

    if(a > 0 && a <= 9){
        printf("satuan");
    }if(a == 0){
        printf("nol");
    }if(a >= 10 && a < 20){
        printf("belasan");
    }if(a >= 20 && a < 100){
        printf("puluhan");
    }if(a > 99){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

}

File: MODUL 3/PYTHON/PRAK304-2310817320006-Alya_Rosaan.py
a = int(input())
if a > 0 and a <= 10:
    print("satuan")
elif a == 0:
    print("nol")
elif a >= 11 and a < 20:
    print("belasan")
elif a >= 20 and a < 100:
    print("puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")
    

